changelog.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 25th January 2021 12:00 UTC
+
+* Added more restriction [use cases] (https://mews-systems.gitbook.io/channel-manager-api/channel-manager-api#restriction-examples).
```

Comment:
```suggestion
* Added more restriction [use cases] (channel-manager-api.md#restriction-examples).
```

Reply:
link should be relative not full path + .md on the end of the file so it works everywhere including github